sentinel-cluster/sentinel-cluster-client-default/src/main/java/com/alibaba/csp/sentinel/cluster/client/NettyTransportClient.java

```diff
@@ -212,6 +218,10 @@ public ClusterResponse sendRequest(ClusterRequest request) throws Exception {
         if (!validRequest(request)) {
             throw new SentinelClusterException(ClusterErrorMessages.BAD_REQUEST);
         }
+        // fix bug https://github.com/alibaba/Sentinel/issues/1906
+        if (!checkParamFlowData(request)){
```

Comment:
`NettyTransportClient` is designed as the transport-layer implementation, which is not associated to the concrete request type (and content). Maybe adding the logic in `DefaultClusterTokenClient` seems better?

Reply:
Moved `checkParamFlowData` to `DefaultClusterTokenClient`. The logic of `checkParamFlowData` has not changed. However, if there are other `ClusterTokenClient` implementations in the future, the `checkParamFlowData` logic cannot be reused.